ui/apps/platform/src/Containers/Integrations/IntegrationsListPage/IntegrationsTable.tsx

```diff
@@ -59,9 +59,9 @@ function IntegrationsTable({
     } = useTableSelection<Integration>(integrations);
     const { isFeatureFlagEnabled } = useFeatureFlags();
 
-    const columns = tableColumnDescriptor[source][type].filter(({ featureFlagDependency }) => {
-        if (typeof featureFlagDependency === 'string') {
-            return isFeatureFlagEnabled(featureFlagDependency);
+    const columns = tableColumnDescriptor[source][type].filter((integration) => {
```

Comment:
Wouldn't it be fine to keep it destructured? What's the additional benefit of not doing it?

Reply:
TypeScript became confused and reported an error:
TS2345: Argument of type 'string' is not assignable to parameter of type 'FeatureFlagEnvVar'.

It surprised me too. Apparently an edge case. TypeScript will probably fix in a future update.

Followed example of tableColumnDescriptor, which did not destructure and did not have the error.